src/DotPulsar/Internal/Abstractions/IMessageTracker.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace DotPulsar.Internal.Abstractions
+{
+    public interface IMessageTracker
```

Comment:
Was this obsoleted by a design change?  Looks unused.

Reply:
good catch